data-prepper-plugins/parquet-codecs/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'java'
+}
+
+
+repositories {
+    mavenCentral()
+    maven { url 'https://jitpack.io'}
```

Comment:
What is this Maven repository for? I think you can remove it.

Reply:
yes it can be removed.